### PR TITLE
Feat/info component

### PR DESCRIPTION
**Pull request by andrereis26**
https://app.clickup.com/t/86byhy6am

Replicated the [gamejam22](https://gamejam2022.nei-isep.org/) but couldn't replicate the animation always playing up on scrolling.

### CODE REVIEW

File: src/data/info.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        items: [
+            <>
+                A terceira edição da LEVEL-UP GAMEJAM <br /> criada pelo NEI regressa. Maior e melhor.
+            </>
+        ]
+    },
+    {
+        title: 'Data',
+        items: [
+            '3 junho (18h) - 5 junho (18h)',
```
Comment (andrefbarros) [inline src/data/info.tsx]: Change this to 24 maio (20h) - 26 maio (16h)

File: src/data/info.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    title: string;
+    items: (string | JSX.Element)[];
+};
+
+export const info: Info[] = [
+    {
+        title: 'Are you ready?',
+        items: [
+            <>
+                A terceira edição da LEVEL-UP GAMEJAM <br /> criada pelo NEI regressa. Maior e melhor.
```
Comment (andrefbarros) [inline src/data/info.tsx]: Change this to: A quinta edição da Gamejam está de volta! Contamos contigo?

File: src/components/Info/InfoContainer.astro [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+import Info from './Info';
+---
+
+<section
+  class="flex flex-col justify-center lg:flex-row lg:space-x-10 mx-auto gap-y-12 pb-12 text-white font-misterPixel"
```
Comment (andrefbarros) [inline src/components/Info/InfoContainer.astro]: gap-y-12 pb-12 i think we can remove one of them

**Review by andrefbarros — CHANGES_REQUESTED** [review]
change the target branch to 'dev'

**Comment by tomasflopes** [comment]
Fixed animation not displaying correctly when in view

File: src/components/Info/Info.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+interface InfoContentProps {
+  children?: React.ReactNode;
+}
+
+const InfoContent: React.FC<InfoContentProps> = () => {
+  return (
+    <div
+      className="flex flex-col"
+      style={{ textShadow: '#000000 2px 2px 0px, #262525 2px 2px 0px' }}>
```
Comment (tomasflopes) [inline src/components/Info/Info.tsx:13]: Make this a custom tailwind style

File: src/components/Info/Info.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+          key={index}
+          className={index % 2 === 0 ? 'col-12 mb-8 text-left' : 'col-12 mb-8 text-right'}
+          initial={{ opacity: 0, x: index % 2 === 0 ? 100 : -100 }}
+          whileInView={{ opacity: 1, x: 0 }}
+          transition={{ delay: index * 0.15 }}
+          viewport={{
+            once: true
+          }}>
+          <h1
+            style={{ textShadow: '#3d1080 2px 2px 0px, #424141 4px 4px 0px' }}
```
Comment (tomasflopes) [inline src/components/Info/Info.tsx:25]: Make this a custom tailwind style

**Review by tomasflopes — CHANGES_REQUESTED** [review]

File: src/components/Info/Info.tsx (hunk 4 above)
Comment (andrereis26) [reply src/components/Info/Info.tsx:13]: tailwindcss doesnt support text shadow, this was the easy (or lazy) way

**Review by andrefbarros — APPROVED** [review]

**Pull request merged**